.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels: [Refactor]
+    target-branch: development
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    labels: [Refactor]
+    target-branch: development
```

Same suggestion here.
```suggestion
    target-branch: development
    open-pull-requests-limit: 1
```